docker_explorer/de.py

```diff
@@ -231,45 +232,44 @@ def Mount(self, container_id, mountpoint):
     container_object = self.GetContainer(container_id)
     container_object.Mount(mountpoint)
 
-  def GetContainersString(self, only_running=False):
-    """Returns a string describing the running containers.
+  def GetContainersJson(self, only_running=False):
+    """Returns a JSON string describing the running containers.
 
     Args:
       only_running (bool): Whether we display only running Containers.
 
     Returns:
-      str: the string displaying information about running containers.
+      dict: A dict object representing the containers.
     """
-    result_string = ''
+    result = []
     for container_object in self.GetContainersList(only_running=only_running):
       image_id = container_object.image_id
       if self.docker_version == 2:
         image_id = image_id.split(':')[1]
+      container_json = {
+          'container_id': container_object.container_id,
+          'image_id': image_id
+      }
 
       if container_object.config_labels:
-        labels_list = ['{0:s}: {1:s}'.format(k, v) for (k, v) in
-                       container_object.config_labels.items()]
-        labels_str = ', '.join(labels_list)
-        result_string += 'Container id: {0:s} / Labels : {1:s}\n'.format(
-            container_object.container_id, labels_str)
-      else:
-        result_string += 'Container id: {0:s} / No Label\n'.format(
-            container_object.container_id)
-      result_string += '\tStart date: {0:s}\n'.format(
-          utils.FormatDatetime(container_object.start_timestamp))
-      result_string += '\tImage ID: {0:s}\n'.format(image_id)
-      result_string += '\tImage Name: {0:s}\n'.format(
-          container_object.config_image_name)
-
-    return result_string
+        container_json['labels'] = container_object.config_labels.items()
```

Comment:
`items()` is a list of tuples, I think this is represented as a "list of lists" in JSON. Is that what you want? Why not just leave the `config_labels` dict as it is?